packages/eslint-config-react/package.json

```diff
@@ -20,18 +20,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@zonedigital/eslint-config-zone": "^2.0.7-beta.0"
-  },
-  "devDependencies": {
-    "eslint": "^4.0.0",
-    "eslint-plugin-compat": "^2.0.0",
-    "eslint-plugin-import": "^2.0.0",
-    "eslint-plugin-react": "^7.0.0"
-  },
-  "peerDependencies": {
-    "eslint": "^4.0.0",
-    "eslint-plugin-compat": "^2.0.0",
-    "eslint-plugin-import": "^2.0.0",
-    "eslint-plugin-react": "^7.0.0"
+    "@zonedigital/eslint-config-zone": "^2.0.7-beta.0",
```

Comment:
`@zonedigital/eslint-config-zone` is now dependant on `eslint-plugin-compat`, `eslint-plugin-import`